core/src/main/java/com/sequenceiq/cloudbreak/controller/OrganizationController.java

```diff
@@ -34,7 +34,7 @@
 public class OrganizationController extends NotificationController implements OrganizationEndpoint {
 
     @Inject
-    @Qualifier("conversionService")
+    @Named("conversionService")
```

Comment:
why do we need this anyway? Bubba couldn't tell for sure. are there more than one conversionService in the application context? why not letting it to resolve it by the variable name?

Reply:
Qualifier does not work with Inject.

Named is required otherwise you got this: org.glassfish.hk2.api.UnsatisfiedDependencyException: There was no object available for injection at SystemInjecteeImpl(requiredType=ConversionService,parent=OrganizationController,qualifiers={},position=-1,optional=false,self=false,unqualified=null,995492775)

Reply:
I see!

Reply:
This is the place where conversionService bean is configured: 
https://github.com/hortonworks/cloudbreak/blob/master/core/src/main/java/com/sequenceiq/cloudbreak/conf/ConversionConfig.java#L14